Packages/tests/Basic/UTF_SweepFormula.ipf

```diff
@@ -2472,76 +2486,20 @@ End
+	string str, trace, key, name, win, device, epoch2, textKey, textValue
 	variable activeChannelsDA = 4
-	variable mode = DATA_ACQUISITION_MODE
-	string channelType = StringFromList(XOP_CHANNEL_TYPE_DAC, XOP_CHANNEL_NAMES)
-
-	string channelTypeC = channelType + "C"
 
 	[win, device] = CreateFakeDataBrowserWindow()
 
-	WAVE/T numericalKeys = GetLBNumericalKeys(device)
-	WAVE numericalValues = GetLBNumericalValues(device)
-	KillWaves numericalKeys, numericalValues
-
-	Make/FREE/T/N=(1, 1) keys = {{channelTypeC}}
-	Make/U/I/N=(numChannels) connections = {7,5,3,1,0}
-	Make/U/I/N=(numSweeps, numChannels) channels = q * 2 // 0, 2, 4, 6, 8 used for all sweeps
-	Make/D/FREE/N=(LABNOTEBOOK_LAYER_COUNT) values = NaN
-
-	Make/FREE/N=(128, numSweeps, numChannels) input = q + p^r
-
-	Make/FREE/T/N=(1, 1) keysEpochs = {{EPOCHS_ENTRY_KEY}}
-	Make/FREE/N=(1, 1, LABNOTEBOOK_LAYER_COUNT)/T wEpochStr
-	wEpochStr = "0.5000000,0.5100000,Epoch=0;Type=Pulse Train;Amplitude=1;Pulse=48;ShortName=E0_PT_P48;,2,:"
-	wEpochStr += "0.5030000,0.5100000,Epoch=0;Type=Pulse Train;Pulse=48;Baseline;ShortName=E0_PT_P48_B;,3,:"
-	wEpochStr += "0.6000000,0.7000000,NoShortName,3,:"
+	textKey   = EPOCHS_ENTRY_KEY
+	textValue = "0.5000000,0.5100000,Epoch=0;Type=Pulse Train;Amplitude=1;Pulse=48;ShortName=E0_PT_P48;,2,:"
+	textValue += "0.5030000,0.5100000,Epoch=0;Type=Pulse Train;Pulse=48;Baseline;ShortName=E0_PT_P48_B;,3,:"
+	textValue += "0.6000000,0.7000000,NoShortName,3,:"
 	epoch2 = "Epoch=0;Type=Pulse Train;Pulse=49;Baseline;"
-	wEpochStr += "0.5100000,0.5200000," + epoch2 + ",2,"
-
-	DFREF dfr = GetDeviceDataPath(device)
-	for(i = 0; i < numSweeps; i += 1)
-		sweepNumber = i
+	textValue += "0.5100000,0.5200000," + epoch2 + ",2,"
 
-		WAVE sweepTemplate = GetDAQDataWave(device, DATA_ACQUISITION_MODE)
-		WAVE sweep = FakeSweepDataGeneratorDefault(sweepTemplate, numChannels)
-		WAVE config = GetDAQConfigWave(device)
-		Redimension/N=(numChannels, -1) config
-
-		for(j = 0; j < numChannels; j += 1)
-			name = UniqueName("data", 1, 0)
-			trace = "trace_" + name
-			Extract input, $name, q == i && r == j
-			WAVE wv = $name
-			AppendToGraph/W=$win wv/TN=$trace
-			channelNumber = channels[i][j]
-			TUD_SetUserDataFromWaves(win, trace, {"experiment", "fullPath", "traceType", "occurence", "channelType", "channelNumber", "sweepNumber"},         \
-									 {"blah", GetWavesDataFolder(wv, 2), "Sweep", "0", channelType, num2istr(channelNumber), num2istr(sweepNumber)})
-			values[connections[j]] = channelNumber
-			config[j][%ChannelType]   = XOP_CHANNEL_TYPE_DAC
-			config[j][%ChannelNumber] = channelNumber
-		endfor
-
-		// create sweeps with dummy data for sweeps() operation thats called when omitting select
-		MoveWave sweep, dfr:$GetSweepWaveName(sweepNumber)
-		MoveWave config, dfr:$GetConfigWaveName(sweepNumber)
-		MIES_DB#DB_SplitSweepsIfReq(win, sweepNumber)
-
-		// channels setup DA: 8, 6, NaN, 4, NaN, 2, NaN, 0, NaN
```

Comment:
I think the comment should also be transferred, as it helps to understand the test.
At first glance this difference between ADC and DAC is not as obvious.